src/autolabel/schema.py

```diff
@@ -172,3 +189,16 @@ class CacheEntry(BaseModel):
 
     class Config:
         orm_mode = True
+
+
+class RefuelLLMResult(BaseModel):
+    """List of generated outputs. This is a List[List[]] because
+    each input could have multiple candidate generations."""
+
+    generations: List[List[Generation]]
+
+    """Errors encountered while running the labeling job"""
+    errors: List[Optional[LabelingError]]
+
+    """Costs incurred during the labeling job"""
+    cost: Optional[float] = 0.0
```

Comment:
Should we track cost individually as well? List[float] ? this is going to be needed for putting it in the row-level metadata anyway